frontend/src/routes/Governance/components/github.min.css

```
@@ -0,0 +1,96 @@
+pre code.hljs {
+  display: block;
+  overflow-x: auto;
+  padding: 1em;
```

We need this so that the "# The difference is redacted because it contains..." message and other long lines are wrapped and not cut off.
```suggestion
  padding: 1em;
  white-space: pre-wrap;
```